chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/ComplaintController.java

```diff
@@ -53,6 +72,110 @@ public ComplaintController(ComplaintManager complaintManager, ErrorMessageUtil e
         return results;
     }
 
+    @Operation(summary = "Search complaints accessible to the logged-in user based on a set of filters.",
+            description = "Security Restrictions: ROLE_ADMIN & ROLE_ONC can get all complaints. "
```

Comment:
ROLE_ONC_STAFF can see this data as well. I checked, and the code _does_ allow access, so I think it's just documentation that ought to be updated